robot_skills/src/robot_skills/api.py

```diff
@@ -35,7 +35,13 @@ def query(self, description, grammar, target, timeout=10):
         if callable(self._pre_hook):
             self._pre_hook()
 
-        answer = self._client.query(description, grammar, target, timeout)
+        try:
+            answer = self._client.query(description, grammar, target, timeout)
+        except TimeoutException as e:
+            rospy.logdebug(e)
```

Comment:
The log is not needed because you re-reraise, the exception can be logged on a higher level

Reply:
Done, please review again.